application/build.gradle.kts

```diff
@@ -1,5 +1,11 @@
 plugins {
     id("bisq.java-library")
+    alias(libs.plugins.openjfx)
```

Comment:
the application module is used in headless appliactions (seed node,...) and we do not want to add a JavaFx dependency to those.